pkg/operator/deploy/deploy.go

```diff
@@ -106,6 +107,18 @@ func (o *operator) resources() ([]runtime.Object, error) {
 						Value: "development",
 					})
 				}
+
+				// define requests and limits for operator pod
+				d.Spec.Template.Spec.Containers[i].Resources = corev1.ResourceRequirements{
```

Comment:
Any reason not to include this in the `pkg/operator/deploy/staticresources/master/deployment.yaml` and `pkg/operator/deploy/staticresources/worker/deployment.yaml` instead of doing this?   
_(Most dynamic content is done this way but being that this isn't going to change like `version.GitCommit`, it maybe better served in the staticresources folder)_

Also, do we want to make sure that the requests/limits make the QOS class "guaranteed" in an effort for scheduling workloads with what cx do?